pkg/util/autopilot.go

```diff
@@ -76,15 +81,17 @@ func IsGKEAutopilotCluster(c client.Client) (bool, error) {
 		return false, nil
 	}
 
-	webhook := &admissionregistrationv1.ValidatingWebhookConfiguration{}
-	objectKey := client.ObjectKey{Name: "policycontrollerv2.config.common-webhooks.networking.gke.io"}
-	err := c.Get(context.Background(), objectKey, webhook)
-	if err == nil {
-		return true, nil
-	} else if apierrors.IsNotFound(err) {
-		return false, nodesErr
+	for _, webhookName := range autopilotWebhooks {
+		webhook := &admissionregistrationv1.ValidatingWebhookConfiguration{}
+		objectKey := client.ObjectKey{Name: webhookName}
+		err := c.Get(context.Background(), objectKey, webhook)
+		if err == nil {
+			return true, nil
+		} else if !apierrors.IsNotFound(err) {
+			return false, err
```

Comment:
The range order would affect the result, if only some of the webhooks are successfully got. We should make the behavior more deterministic.

Reply:
Updated with a multierror to collect the apiserver errors.